src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
The link rel attribute should reference a CSS file, not SCSS. Change the href to './styles/main.css'